.github/workflows/release.yml

```diff
@@ -28,10 +32,38 @@ jobs:
         id: build_vars
         uses: ./.github/actions/build-vars
 
+  disable_functions:
+    name: "Set function ${{ matrix.function }} disabled"
+    needs:
+      - pre_job
+    if: |
+      needs.pre_job.outputs.has_router_change == 'true' ||
+      needs.pre_job.outputs.has_react_change == 'true'
```

Comment:
I *think* we only need `has_router_change` here but it might be worth pinging the #prime-reportstream-engineering channel to see.